lib/tss/response_handlers.rb

```diff
@@ -16,4 +16,12 @@ def handle_get_response(response, path, root_name)
     else fail "TSS: 'GET #{path}' failed with status = #{response.code}."
     end
   end
+
+  def handle_put_response(response, path)
+    case response.code
+    when 200 then JSON.parse(response.body)
+    when 404 then nil
+    else fail "TSS: 'PUT #{path}' failed with status = #{response.code}."
+    end
+  end
```

Comment:
I don't think we'd have any advantage from checking the exact value. Checking that the response is between 200 and 300 would be sufficient. So probably something like this:

```ruby
def handle_response(verb, response, path, root_name = nil)
     fail "TSS: '#{verb} #{path}' failed with status = #{response.code}." unless response.code.between?(200, 299)
     response_json = JSON.parse(response.body)
     root_name.nil? ? response_json : response_json[root_name]
   end
```